src/openforms/submissions/tests/test_views.py

```diff
@@ -0,0 +1,371 @@
+from unittest.mock import patch
+
+from django.http import HttpRequest
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+from furl import furl
+
+from openforms.accounts.tests.factories import UserFactory
+from openforms.authentication.constants import FORM_AUTH_SESSION_KEY
+from openforms.submissions.tests.factories import SubmissionFactory
+
+
+class SearchSubmissionForCosignView(TestCase):
```

Comment:
wouldn't it make more sense to use `WebTest` here instead of Django's bare test case?

Also - why are there `self.client.login(...)` calls here - shouldn't this be used by non-staff users so anonymous Django users?